.github/policies/resourceManagement.yml

```diff
@@ -0,0 +1,34 @@
+name: GitOps.PullRequestIssueManagement
+description: GitOps.PullRequestIssueManagement primitive
+owner: null
+resource: repository
+disabled: false
```

Comment:
Should this be set? I wonder if the default value of `disabled` is `false`, in which case you could probably remove it.